Tests/PKHeX.Core.Tests/Simulator/ShowdownSetTests.cs

```diff
@@ -17,160 +17,151 @@ static ShowdownSetTests()
+            Assert.True(encs.Count > 0);
+            Assert.True(encs.All(z => z.Species > 150));
 
             var info = new SimpleTrainerInfo();
             var enc = encs[0];
             var pk = enc.ConvertToPKM(info);
 
             var la = new LegalityAnalysis(pk);
-            Assert.IsTrue(la.Valid);
+            Assert.True(la.Valid);
         }
 
-        [TestMethod]
-        [TestCategory(SimulatorParse)]
+        [Fact]
         public void SimulatorGetVCEgg1()
         {
             var set = new ShowdownSet(SetSlowpoke12);
             var pk7 = new PK7 { Species = set.Species, AltForm = set.FormIndex, Moves = set.Moves, HT_Name = "PKHeX" };
             var encs = EncounterMovesetGenerator.GenerateEncounters(pk7, set.Moves, GameVersion.GD).ToList();
-            Assert.IsTrue(encs.Count > 0);
+            Assert.True(encs.Count > 0);
 
             var info = new SimpleTrainerInfo();
             var enc = encs[0];
             var pk = enc.ConvertToPKM(info);
 
             var la = new LegalityAnalysis(pk);
-            Assert.IsTrue(la.Valid);
+            Assert.True(la.Valid);
         }
 
-        [TestMethod]
-        [TestCategory(SimulatorParse)]
+        [Fact]
         public void SimulatorGetSmeargle()
         {
             var set = new ShowdownSet(SetSmeargle);
             var pk7 = new PK7 { Species = set.Species, AltForm = set.FormIndex, Moves = set.Moves };
             var encs = EncounterMovesetGenerator.GenerateEncounters(pk7, set.Moves, GameVersion.MN);
-            Assert.IsTrue(encs.Any());
+            Assert.True(encs.Any());
             encs = EncounterMovesetGenerator.GenerateEncounters(pk7, set.Moves, GameVersion.MN);
             var first = encs.FirstOrDefault();
-            Assert.IsTrue(first != null);
+            Assert.NotNull(first);
 
             var enc = first;
             var info = new SimpleTrainerInfo();
             var pk = enc.ConvertToPKM(info);
 
             var la = new LegalityAnalysis(pk);
-            Assert.IsTrue(la.Valid);
+            Assert.True(la.Valid);
         }
 
-        [TestMethod]
-        [TestCategory(SimulatorParse)]
+        [Fact]
         public void SimulatorParseMultiple()
         {
             var text = string.Join("\r\n\r\n", Sets);
             var lines = text.Split(new[] {"\r\n", "\n"}, StringSplitOptions.None);
             var sets = ShowdownSet.GetShowdownSets(lines);
-            Assert.IsTrue(sets.Count() == Sets.Length);
+            Assert.True(sets.Count() == Sets.Length);
 
             sets = ShowdownSet.GetShowdownSets(Enumerable.Empty<string>());
-            Assert.IsTrue(!sets.Any());
+            Assert.True(!sets.Any());
 
             sets = ShowdownSet.GetShowdownSets(new [] {"", "   ", " "});
-            Assert.IsTrue(!sets.Any());
+            Assert.True(!sets.Any());
         }
 
-        //[TestMethod]
-        //[TestCategory(SimulatorParse)]
+        [Fact]
```

Comment:
I could make this a theory, so we have one test per loop iteration. This way, we know how many things fail when there's at least one failure.